apps/lockfile-explorer-server/package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "@rushstack/lockfile-explorer-server",
+  "description": "(Copy of sample project) Building this project is a regression test for Heft",
+  "version": "1.0.0",
+  "private": true,
+  "main": "lib/index.js",
+  "license": "MIT",
+  "scripts": {
+    "build": "heft build --clean",
+    "_phase:build": "heft build --clean",
+    "_phase:test": "heft test --no-build",
+    "start": "nodemon ./src/index.ts"
+  },
+  "devDependencies": {
+    "@rushstack/eslint-config": "workspace:*",
+    "@rushstack/heft": "workspace:*",
+    "@rushstack/heft-node-rig": "workspace:*",
+    "@types/heft-jest": "1.0.1",
+    "@types/node": "12.20.24",
+    "ts-node-dev": "~2.0.0",
```

Comment:
It'd be preferable to use the compiled output from Heft instead of ts-node.

Reply:
changed to use heft